### PR TITLE
Release 0.4.0 details

### PR DESCRIPTION
**Pull request by mangelajo**
Updates the release history with the 0.4.0 details

[Needs SQUASH to provide a proper commit title]

### CODE REVIEW

File: src/content/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,32 @@ pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.4.0 Libreswan cable driver, k8s multicluster service discovery
+
+> This release is focused on the libreswan cabledriver implementation, as well 
+> as standardizing the service discovery support in lighthouse with the upstream
+> kubernetes kep [1].
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:12]:
```suggestion
> This release is mainly focused on the libreswan Submariner cable driver implementation, as well 
> as standardizing the service discovery support in Lighthouse with the upstream
> kubernetes KEP [1].
```

File: src/content/releases/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,32 @@ pre = "<b>5. </b>"
...
+
+* The **libreswan** IPsec cabledriver is available for testing. It's been included
+  on the testing matrix of the project.
+* **Lighthouse** behavior has been modified following [1] for the next details:
+   - A `ServiceExport` object needs to be created alongside any `Service` that we
+     want exported to the participant clusters
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:18]:
```suggestion
* **Lighthouse** has been modified per [1] as follows:
   - A `ServiceExport` object needs to be created alongside any `Service` that is intended to be exported to participant clusters
```

File: src/content/releases/_index.en.md [hunk 3]
```diff
@@ -5,6 +5,32 @@ pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.4.0 Libreswan cable driver, k8s multicluster service discovery
```
Comment (nyechiel) [inline src/content/releases/_index.en.md:8]: Change k8s to Kubernetes

File: src/content/releases/_index.en.md [hunk 4]
```diff
@@ -5,6 +5,32 @@ pre = "<b>5. </b>"
...
+
+> This release is focused on the libreswan cabledriver implementation, as well 
+> as standardizing the service discovery support in lighthouse with the upstream
+> kubernetes kep [1].
+
+* The **libreswan** IPsec cabledriver is available for testing. It's been included
```
[the remark this answers is not in the document]
Comment (nyechiel) [reply src/content/releases/_index.en.md:14]: s/libreswan/Libreswan

File: src/content/releases/_index.en.md [hunk 5]
```diff
@@ -5,6 +5,32 @@ pre = "<b>5. </b>"
...
+  on the testing matrix of the project.
+* **Lighthouse** behavior has been modified following [1] for the next details:
+   - A `ServiceExport` object needs to be created alongside any `Service` that we
+     want exported to the participant clusters
+   - The supercluster services can now be accessed with `<service-name>.<namespace>.svc.supercluster.local`
+
+* **Globanet** overlapping CIDR support improvement and bugfixes
```
[the remark this answers is not in the document]
Comment (nyechiel) [reply src/content/releases/_index.en.md:21]: @sridhargaddam anything in particular we should highlight? At the minimum we should use "improvements" and not just one "improvement"

File: src/content/releases/_index.en.md (hunk 3 above)
Comment (mangelajo) [inline src/content/releases/_index.en.md:8]:
```suggestion
## v0.4.0 Libreswan cable driver, Kubernetes multicluster service discovery
```

File: src/content/releases/_index.en.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (sridhargaddam) [reply src/content/releases/_index.en.md:21]: yes agree @nyechiel, there were couple of improvements mostly around stability.

Having said it, we still have some known issues that needs to be handled and until then its possible that CI would occasionally fail. 
i.e., 
1. we have to update the e2e connector pod logic to wait until its Ready/globalIp-annotated before starting the tests.
2. the redundancy tests in e2e are only looking at submariner-gateway pod status and not submariner-globalnet status when gateway is migrated. This is occasionally causing some failures.
3. sometimes we are seeing errors in kube-proxy which would affect the functionality of cluster-ip services on the gateway node. I discussed this with @mangelajo and we wanted to try with some latest kube-proxy code (as the one used in KIND is very old).

File: src/content/releases/_index.en.md [hunk 6]
```diff
@@ -5,6 +5,31 @@ pre = "<b>5. </b>"
...
+> as standardizing the service discovery support in Lighthouse with the upstream
+> kubernetes KEP [1].
+
+* The **libreswan** IPsec cable driver is available for testing. It's been included
+  on the testing matrix of the project.
+* **Lighthouse** has been modified per [1] as follows:
+   - A `ServiceExport` object needs to be created alongside any `Service` that is intended to be exported to participant clusters
+   - The supercluster services can now be accessed with `<service-name>.<namespace>.svc.supercluster.local`
+
+* **Globanet** overlapping CIDR support improvement and bug fixes
```
Comment (mangelajo) [inline src/content/releases/_index.en.md:20]:
```suggestion
* **Globanet** overlapping CIDR support improvements and bug fixes
```

**Pull request merged**